operators/endpointmetrics/manifests/prometheus/kube-state-metrics-clusterRoleBinding.yaml

```diff
@@ -9,4 +9,4 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: kube-state-metrics
-  namespace: open-cluster-management-addon-observability
+  namespace: open-cluster-management-observability
```

Comment:
is it ok to hardcode change all these now or is it just for the purposes of getting the tests passed?